src/views/EdgeFirewall/FormFields/FormFieldsEdgeFirewall.vue

```diff
@@ -132,18 +133,18 @@
           </template>
         </PickList>
         <small class="text-color-secondary text-sm font-normal leading-tight">
-          Hold command or ctrl to select multiple items.
+          Select an item from the list and then use the arrows to move it between the available and
```

Comment:
Choose an item from the list and then use the arrows to move it between the available and selected domains boxes.

Reply:
Nesse caso vou deixar o termo como está porque foi como a Hannah definiu 😭 acho que esse componente aparece em outras telas tbm